demos/simple-gotracing/http_trace/http_trace.go

```diff
@@ -0,0 +1,107 @@
+package main
+
+import "C"
+import (
+	"bytes"
+	"encoding/binary"
+	"flag"
+	"fmt"
+	"os"
+	"os/signal"
+	"unsafe"
+
+	"github.com/fatih/color"
+	"github.com/iovisor/gobpf/bcc"
+)
+
+/*
+#cgo CFLAGS: -I/usr/include/bcc/compat
+#cgo LDFLAGS: -lbcc
+#include <bcc/bcc_common.h>
+#include <bcc/libbpf.h>
+#include <bcc/bcc_syms.h>
+*/
```

Comment:
These chunk of code need comments though.

I have no clue on what purpose they serve. Do they have special purpose outside of being plain comments?
Ignore this comment if this is well-known technique in Golang.